CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/CollectionRequestHandler.swift

```diff
@@ -0,0 +1,139 @@
+//RequestHandler.swift
+//CBLTestServer-iOS
+
+import Foundation
+import CouchbaseLiteSwift
+
+public class CollectionRequestHandler {
+    public func handleRequest(method: String, args: Args) throws -> Any? {
+        switch method {
+        case "collection_defaultCollection":
+            let database: Database = (args.get(name:"database"))!
+            let collection: Collection = try (database.defaultCollection())!
```

Comment:
Forced unwrapping an optional value. This will crash in case default collection is empty.

wait I thought I have already reviewed this is in other PR.

Reply:
?